extutils/maskfinder/url.py

```diff
@@ -1,10 +1,32 @@
+"""
```

Comment:
Codacy found an issue: [One-line docstring should fit on one line with quotes (found 3) (D200)](https://app.codacy.com/manual/RaenonX/Jelly-Bot/pullRequest?prid=6015490)